app/controllers/home_controller.rb

```diff
@@ -1,3 +1,4 @@
 class HomeController < ApplicationController
+  layout "landing"
```

Comment:
this is good

Reply:
+ create an empty pages for Pricing, etc
Just empty page with `h1 Pricing` for example